File: Python/numeroMenor.py
# Elaborar un algoritmo que encuentre el número menor entre 4 números.

ingresaPrimerNumero = float(input("Ingresa el primer número: "))
ingresaSegundoNumero = float(input("Ingresa el segundo número: "))
ingresaTercerNumero = float(input("Ingresa el tercer número: "))
ingresaCuartoNumero = float(input("Ingresa el cuarto número: "))

if (ingresaPrimerNumero < ingresaSegundoNumero) and (ingresaPrimerNumero < ingresaTercerNumero) and (ingresaPrimerNumero < ingresaCuartoNumero):
    print (f"El primer número es el menor {ingresaPrimerNumero:.0f}.")

elif (ingresaSegundoNumero < ingresaPrimerNumero) and (ingresaSegundoNumero < ingresaTercerNumero) and (ingresaSegundoNumero < ingresaCuartoNumero):
    print (f"El segundo número es el menor {ingresaSegundoNumero:.0f}.")

elif ((ingresaTercerNumero < ingresaPrimerNumero) and (ingresaTercerNumero < ingresaSegundoNumero) and (ingresaTercerNumero < ingresaCuartoNumero)):
    print (f"El tercer número es el menor {ingresaTercerNumero:.0f}")

else:
    print(f"El cuarto número es el menor {ingresaCuartoNumero:.0f}")

File: C++/numeroMenor.cpp
/*
Elaborar un algoritmo que encuentre el número menor entre 4 números.
*/

#include <iostream>
using namespace std;

int main()
{
    int ingresePrimerNumero, ingreseSegundoNumero, ingreseTercerNumero, ingreseCuartoNumero;

    cout << "Ingrese primer número: ";
    cin >> ingresePrimerNumero;

    cout << "Ingrese segundo número: ";
    cin >> ingreseSegundoNumero;

    cout << "Ingrese tercer número: ";
    cin >> ingreseTercerNumero;

    cout << "Ingrese cuarto número: ";
    cin >> ingreseCuartoNumero;

    if ((ingresePrimerNumero < ingreseSegundoNumero) && (ingresePrimerNumero < ingreseTercerNumero) && (ingresePrimerNumero < ingreseCuartoNumero))
    {
        cout << "El primer número es el menor de los cuatro: " << ingresePrimerNumero;
    }
    else if ((ingreseSegundoNumero < ingresePrimerNumero) && (ingreseSegundoNumero < ingreseTercerNumero) && (ingreseSegundoNumero < ingreseCuartoNumero))
    {
        cout << "El segundo número es el menor de los cuatro: " << ingreseSegundoNumero;
    }
    else if ((ingreseTercerNumero < ingresePrimerNumero) && (ingreseTercerNumero < ingreseSegundoNumero) && (ingreseTercerNumero < ingreseCuartoNumero))
    {
        cout << "El tercer número es el menor de los cuatro: " << ingreseTercerNumero;
    }
    else
    {
        cout << "El cuarto número es el menor de los cuatro: " << ingreseCuartoNumero;
    }
}